crates/cext/src/transpiler/target.rs

```diff
@@ -570,13 +580,14 @@ pub extern "C" fn qk_target_entry_new_reset() -> *mut TargetEntry {
 ///
 /// @param operation The ``QkGate`` whose properties this target entry defines.
 /// @param params A pointer to the parameters that the instruction is calibrated for.
+/// @param name An optional name for the instruction in the target.
```

Comment:
We might want to comment in the documentation about what the default name would be, if none is specified.